TelligentInstall/VersionInfo.ps1

```diff
@@ -1,5 +1,5 @@
 ﻿$versionDllNames = [ordered]@{
-    Platform = 'Telligent.Evolution.Components.dll', 'CommunityServer.Components.dll'
+    Platform = 'Telligent.Evolution.Platform.dll', 'Telligent.Evolution.Components.dll', 'CommunityServer.Components.dll'
```

Comment:
Agreed, I just removed some of the legacy solr stuff too. I'll remove anything as I find it.